packages/core/test/components/DataTable.test.tsx

```diff
@@ -466,7 +558,7 @@ describe('<DataTable /> handles edits', () => {
   });
 
   it('should enable instant edit mode', () => {
-    const wrapper = shallow(<DataTable {...props} instantEdit />).dive();
+    const wrapper = shallow(<DataTable {...props} />).dive();
```

Comment:
should have `instantEdit` on it? not seeing it defined in `props` as `true`

Reply:
That was an unnecessary prop in the previous version, `instantEdit` is and always was default `true`

Reply:
aah, gotcha!